cat-launcher/src-tauri/src/active_release/commands.rs

```diff
@@ -0,0 +1,43 @@
+use serde::{ser::SerializeStruct, Serializer};
+use strum::IntoStaticStr;
+use tauri::{command, State};
+
+use crate::active_release::active_release::ActiveReleaseError;
+use crate::active_release::repository::sqlite_active_release_repository::SqliteActiveReleaseRepository;
+use crate::variants::GameVariant;
+
+#[derive(thiserror::Error, Debug, IntoStaticStr)]
+pub enum ActiveReleaseCommandError {
+    #[error("failed to get active release: {0}")]
+    GetActiveRelease(#[from] ActiveReleaseError),
+
+    #[error("failed to get system directory: {0}")]
+    SystemDirectory(#[from] tauri::Error),
```

Comment:
In cat-launcher/src-tauri/src/active_release/commands.rs around lines 14-15, the
error enum contains an unused SystemDirectory(#[from] tauri::Error) variant;
remove that variant from the enum (and any #[from] conversion attribute), then
remove any now-unused tauri::Error imports or use statements and run cargo check
to ensure no remaining references; if you intend to keep it for planned future
use instead, annotate it with #[allow(dead_code)] and leave a comment explaining
why.